### PR TITLE
clarify feed content rules and subissiom acceptance criterias

### PR DESCRIPTION
**Pull request by matthiasjost**
clarify feed content rules and submission acceptance criteria

### CODE REVIEW

**Comment by matthiasjost** [comment]
@DeeJayTC Please see if that fits our needs.

File: creator-cards/_creatorTemplate.json [hunk 1]
```diff
@@ -13,23 +13,19 @@
...
-  "channels": [
+  "channels": [ // You must have published at least three post (long form content) since 2022 on any of your channels. About .NET or SPA frontend technologies
     {
-      "name": "Site | Twitch | Youtube | etc",
-      "url": "https://ilovedotnet.org"
+      "name": "Blog|Site|Twitch|Youtube|Book|etc",
+      "url": "https://www.wearedotnet.io/"
     }
   ],
-  "feeds": [
```
Comment (DeeJayTC) [inline creator-cards/_creatorTemplate.json:22]: lets please stick to the feeds array and just use the type. 
currently the type isn't used at all. We could just change that to these types:

dotnet, 
personal,


any dotnet feed must follow the rules, personal is only visible on profile page but can contain other topics as well

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -26,6 +33,17 @@
...
+
+* .NET itself
+* .NET libraries
+* .NET languages C#, F#, Visual Basic
+* Tools needed during development (for .NET)
+* CI/CD
+* SPA Frontend technologies like Angular, Vue.js, React etc.
+
+* [ ] the content under feedsOffTopic may contain off-topic posts or posts indirectly related to .NET like frontend-topics, database etc.
+
```
Comment (DeeJayTC) [inline .github/PULL_REQUEST_TEMPLATE.md:46]: Missing things like Azure services, Azure Hosting or any other topic thats totally related to .NET

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -26,6 +33,17 @@
...
+* [ ] The content under feedsDotnet must be pure .NET meaning:
+
+* .NET itself
+* .NET libraries
+* .NET languages C#, F#, Visual Basic
+* Tools needed during development (for .NET)
+* CI/CD
+* SPA Frontend technologies like Angular, Vue.js, React etc.
+
+* [ ] the content under feedsOffTopic may contain off-topic posts or posts indirectly related to .NET like frontend-topics, database etc.
```
Comment (DeeJayTC) [inline .github/PULL_REQUEST_TEMPLATE.md:45]: see below

**Comment by matthiasjost** [comment]
@DeeJayTC Please check if it's now according to what is needed.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -26,6 +34,20 @@
...
+* [ ] The content with feed.type = dotnet must be .NET related:
+* .NET itself
+* .NET libraries
+* .NET languages C#, F#, Visual Basic
+* Tools needed during development (for .NET)
+* CI/CD
+* SPA Frontend technologies like Angular, Vue.js, React etc.
+* Cloud services related to .NET applications (AWS, Azure...)
+
+* [ ] the content under feed.ype = personal may contain off-topic posts.
```
Comment (DeeJayTC) [inline .github/PULL_REQUEST_TEMPLATE.md:46]: typo

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -26,6 +34,20 @@
 
 ## RSS or Atom Feed 
 
+* [ ] The content with feed.type = dotnet must be .NET related:
+* .NET itself
+* .NET libraries
+* .NET languages C#, F#, Visual Basic
+* Tools needed during development (for .NET)
+* CI/CD
+* SPA Frontend technologies like Angular, Vue.js, React etc.
```
Comment (DeeJayTC) [inline .github/PULL_REQUEST_TEMPLATE.md:43]: add: "in combinaton with .NET" we dont want pure angular or react stuff if theres not at least some relation to .net

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -26,6 +34,20 @@
 
 ## RSS or Atom Feed 
 
+* [ ] The content with feed.type = dotnet must be .NET related:
+* .NET itself
+* .NET libraries
```
Comment (DeeJayTC) [inline .github/PULL_REQUEST_TEMPLATE.md:39]: .NET libraries and projects

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -15,6 +15,14 @@
 ## When Do We Accept Someone's Creator Card Submission? 
 
 * [ ] You must have published at least three post (long form content) since 2022 on any of your channels.
```
Comment (DeeJayTC) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: replace post with just "content pieces" maybe?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -26,6 +34,20 @@
...
+* Cloud services related to .NET applications (AWS, Azure...)
+
+* [ ] the content under feed.ype = personal may contain off-topic posts.
+* Project management
+* leadership
+* Other
+
 * [ ] Submit an Atom or RSS URL inside your creator card that belongs to your blog 
 
 * [ ] If you do not submit an RSS or Atom URL. We cannot consider your feed for the upcoming aggregation feature.
```
Comment (DeeJayTC) [inline .github/PULL_REQUEST_TEMPLATE.md:53]: remove "upcoming"

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (matthiasjost) [reply .github/PULL_REQUEST_TEMPLATE.md:46]: Fixed.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (matthiasjost) [reply .github/PULL_REQUEST_TEMPLATE.md]: fixed.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (matthiasjost) [reply .github/PULL_REQUEST_TEMPLATE.md]: Resolved.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 8 above)
Comment (matthiasjost) [reply .github/PULL_REQUEST_TEMPLATE.md]: Fixed.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 7 above)
Comment (matthiasjost) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done.

**Comment by matthiasjost** [comment]
@DeeJayTC I think we have a pretty good version now. The type for the feed was initially used by my crawler which parsed the URLs from the channels automatically. I don't see any use for it know. So we can easily use the type for personal | dotnet.

**Pull request merged**